README.md

```diff
@@ -94,6 +94,104 @@ class StorageFile():
     def signed_url(version: str, action: str, expires: int, contentType: str) -> str:
 ```
 
+#### Embedded SQL support
```

Comment:
Can we move this content to "docs" and have a short section here like we did in the Node.js example?